test/unit/specs/components/ResultsPage.spec.js

```diff
@@ -4,7 +4,7 @@ import { shallow, createLocalVue } from '@vue/test-utils';
 import ResultsPage from '@/components/ResultsPage';
 import PdfRenderer from '@/utils/pdf-renderer';
 import isExportDocument from '@/utils/is-export-document';
-import exampleResponses from '../../example-responses';
+import exampleTerms from '../../example-terms';
```

Comment:
I notice that you renamed this file but did not change the const declaration and export. It still reads exampleResponses and should probably be renamed for consistency.

Reply:
Good point. I have renamed the constant to match.